de_templates/test_config_ingest.yaml

```diff
@@ -0,0 +1,38 @@
+pipeline_name: Ingest_AzureSql_Example
+pipeline_description: "Ingest from demo Azure SQL database using ingest config file."
+data_source_type: azure_sql
+
+# Used to create an ADF dataset and a linked service, named ds_<dataset_name> and ls_<dataset_name>
+dataset_name: AzureSql_Example
```

Comment:
Could this be use this to make the pipeline name too (i.e. prefix `Ingest_`)?

Reply:
done